### ISSUE TITLE
No Issue

### ISSUE THREAD

**Issue by Teicu** [augustd/burp-suite-swaggy#5]
Hi  

Can you please share even a compiled version? 

Thanks

**Comment by augustd** [comment]
Are you able to build it? Just run `mvn install` in the project root directory.

**Comment by Teicu** [comment]
Hi 

Unfortunately I don't know, I've never done this before. This is why I asked.. 

Regards

**Comment by fuckyur** [comment]
Once packaged, you can add a plug-in to burpsuite, but it doesn't seem to work

**Comment by augustd** [comment]
I was just able to pull the latest code and compile with `mvn install`. 

After build, you will find the binary in `<project root>/target/burp-suite-swaggy-1.0-SNAPSHOT.jar`

**Comment by fuckyur** [comment]
> 我只是能够提取最新的代码并使用进行编译`mvn install`。
> 
> 构建完成后，您将在 `<project root>/target/burp-suite-swaggy-1.0-SNAPSHOT.jar`

Yes, I can find burp-suite-swaggy-1.0 in the / target directory- SNAPSHOT.jar However, when I added it to burp, the WADL type data package was not displayed after it was sent to the plug-in. I don't know if it is my problem
